### PR TITLE
relax deps to allow phoenix 1.6

### PR DESCRIPTION
**Pull request by jaimeiniesta**

### CODE REVIEW

**Comment by jrissler** [comment]
Love to see this merged in!

**Comment by sushant12** [comment]
is this project abandoned? Why is this not merged yet?

File: mix.exs [hunk 1]
```diff
@@ -49,7 +49,7 @@ defmodule ScrivenerHtml.Mixfile do
     [
       {:scrivener, "~> 1.2 or ~> 2.0"},
       {:phoenix_html, "~> 2.2"},
-      {:phoenix, "~> 1.0 and < 1.5.0", optional: true},
+      {:phoenix, "~> 1.0 and < 1.6.0", optional: true},
```
Comment (hkrutzer) [inline mix.exs:52]:
```suggestion
      {:phoenix, "~> 1.0", optional: true},
```
I don't think it's necessary to keep this constraint

**Comment by blaza1** [comment]
@mgwidmann scrivener pagination is really useful for our application but it's blocking us upgrading our application to phoenix 1.5. Can we get this merged in ?

**Comment by btbinhtran** [comment]
Just sent @mgwidmann an email. Hopefully he responds and maybe we can get this merged in.

**Comment by williamweckl** [comment]
Any updates on this?

**Comment by yellow5** [comment]
Noting that #98 is also trying to add support for Phoenix v1.5.x

**Comment by benedikt7** [comment]
Are there any updates on this? Still the blocker in upgrading to 1.5.x.....

**Comment by thbar** [comment]
@jaimeiniesta did you have good success just relaxing the dependencies, or is there more to be done.

The PR at https://github.com/mgwidmann/scrivener_html/pull/98/files shows that some tests had to be adapted, but nothing critical.

**Comment by jaimeiniesta** [comment]
@thbar yes, I'm using my branch on a Phoenix 1.5.4 app and it works just fine.

**Comment by thiagomajesk** [comment]
@mgwidmann Anything preventing this from being merged?

PS.: For people upgrading to phoenix 1.5, just add `override: true` in the meanwhile.

**Comment by thbar** [comment]
FYI, I am trying to spread the word in order to find a way to reach Matt, and maybe find a new team of maintainers (even just for simple stuff):

https://github.com/mgwidmann/scrivener_html/issues/103#issuecomment-924269872

**Review by williamweckl — APPROVED** [review]

**Comment by thbar** [comment]
Hello @jaimeiniesta! Can you confirm you are using this PR in production? Thanks!

@williamweckl you have approved the changes, but do you have merge rights on the repo?

**Comment by jaimeiniesta** [comment]
@thbar yes, I'm using this PR in production with Phoenix 1.6.2, Elixir 1.12.3 and it works fine!

**Comment by jaimeiniesta** [comment]
🎉 thanks!

**Pull request merged**